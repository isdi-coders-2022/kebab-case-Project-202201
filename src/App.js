import Header from "./components/Header/Header";
import Streamer from "./components/Streamer/Streamer";

const streamerInfo = {
  id: "124422593",
  display_name: "LEC",
  type: "online",
  broadcaster_type: "partner",
  description:
    "Official Twitch Channel of the League of Legends European Championship. For more information, schedules and stats head to Lolesports.com",
  profile_image_url:
    "https://static-cdn.jtvnw.net/jtv_user_pictures/41b23352-5c99-4eb2-974b-1e9b8120425e-profile_image-300x300.png",
};

function App() {
  return (
    <div>
      <Header />
      <Streamer props={streamerInfo} />
      <Streamer props={streamerInfo} />
    </div>
  );
}

export default App;
