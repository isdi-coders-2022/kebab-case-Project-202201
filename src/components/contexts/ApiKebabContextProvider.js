import { useState } from "react";
import ApiKebabContext from "./ApiKebabContext";

const FormContextProvider = ({ children }) => {
  const [name, setName] = useState("");
  const [imageImput, setImageImput] = useState("");
  const [description, setDescription] = useState("");

  return (
    <ApiKebabContext.Provider
      value={{
        name,
        setName,
        imageImput,
        setImageImput,
        description,
        setDescription,
      }}
    >
      {children}
    </ApiKebabContext.Provider>
  );
};
export default FormContextProvider;
