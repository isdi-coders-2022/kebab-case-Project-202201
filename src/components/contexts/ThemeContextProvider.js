import ThemeContext from "./ThemeContext";

const ThemeContextProvider = ({ children }) => {
  const initialValue = "light";
  return <ThemeContext value={{ initialValue }}>{children}</ThemeContext>;
};
export default ThemeContextProvider;
