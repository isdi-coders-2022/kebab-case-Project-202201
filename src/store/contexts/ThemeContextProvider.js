import ThemeContext from "./ThemeContext";

const ThemeContextProvider = ({ children }) => {
  const initialValue = "light";
  return (
    <ThemeContext.Provider value={{ initialValue }}>
      {children}
    </ThemeContext.Provider>
  );
};
export default ThemeContextProvider;
