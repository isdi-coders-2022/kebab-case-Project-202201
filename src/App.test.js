import App from "./App";
import TestRenderer from "react-test-renderer";
import ThemeContextProvider from "./store/contexts/ThemeContextProvider";
import ApiKebabContextProvider from "./store/contexts/ApiKebabContextProvider";

describe("Given app component", () => {
  describe("When it gets a const app that creates a wrapped app", () => {
    test("then it should create and compare a snapshot component", () => {});
    const app = TestRenderer.create(
      <ThemeContextProvider>
        <ApiKebabContextProvider>
          <App />
        </ApiKebabContextProvider>
      </ThemeContextProvider>
    );
    expect(app.toJSON()).toMatchSnapshot();
  });
});
