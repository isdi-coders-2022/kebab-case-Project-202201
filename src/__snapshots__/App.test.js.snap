// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <header
    className="sc-gsDKAQ lgLrrS"
  >
    <button
      className="sc-bdvvtL cgHFUT"
      onClick={[Function]}
      type="button"
    >
      <img
        alt="logo shawarma"
        src="img/logo.png"
      />
    </button>
    <img
      alt="streaming header title"
      className="sc-dkPtRN eSHNya"
      src="img/streaminglight.png"
    />
    <img
      alt="streaming header title"
      className="sc-hKwDye fVvAZv"
      src="img/shawarmalight.png"
    />
  </header>
  <div
    className="sc-jRQBWg issXiZ"
  >
    <button
      className="sc-eCImPb cJVRaK"
      onClick={[Function]}
      type="button"
    >
      HOME
    </button>
    <button
      className="sc-bdvvtL cgHFUT"
      onClick={[Function]}
      type="button"
    >
      <img
        alt="multidurum"
        src="img/multidurum.png"
      />
    </button>
    <button
      className="sc-eCImPb cJVRaK"
      onClick={[Function]}
      type="button"
    >
      FAVS
    </button>
  </div>
  <article
    className="sc-gKclnd hQnfJi"
  >
    <header
      className="sc-iCfMLu bngWWC"
    >
      <img
        alt="Wololoo"
        src="https://media.kingston.com/hyperx/bio/hx-bio-influencer-dendi.jpg"
      />
      <div>
        <p>
          online/offline
        </p>
        <div
          className="streamer-name-block"
        >
          <h2>
            Wololoo
          </h2>
          <button>
            edit button
          </button>
        </div>
      </div>
      <button>
        shish-kebab
      </button>
    </header>
    <div>
      Reel Container
    </div>
  </article>
</div>
`;
